File: pagelib/src/main/java/cc/zgeek/pagelib/Annotation/DisableInjectView.java
package cc.zgeek.pagelib.Annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by ZGeek.
 * Change History:
 * 2017/1/15 : Create
 */

@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.TYPE)
public @interface DisableInjectView {
}

File: pagelib/src/main/kotlin/cc/zgeek/pagelib/Annotation/DisableInjectView.kt
package cc.zgeek.pagelib.Annotation

import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Created by ZGeek.
 * Change History:
 * 2017/1/15 : Create
 */

@kotlin.annotation.Retention(AnnotationRetention.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class DisableInjectView
